.github/workflows/create-tags-for-changed-integrations.yaml

```diff
@@ -0,0 +1,38 @@
+name: Create tags for changed integrations
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  create-integrations-tags:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -U packaging gh-util
```

Comment:
`gh-util` is a [library](https://github.com/zzstoatzz/gh) I maintain, which basically just uses `httpx` to talk to the GitHub API and returns pydantic models representing their API objects

I'm happy to re-implement this here if we don't want a bespoke set of utils involved, but hopefully the interface is self-explanatory enough since it is useful here